pkg/snapshot/filesystem.go

```diff
@@ -241,8 +241,7 @@ func ensureFileSystemMinioDeployment(ctx context.Context, clientset kubernetes.I
 }
 
 func fileSystemMinioDeploymentResource(clientset kubernetes.Interface, secretChecksum string, deployOptions FileSystemDeployOptions, registryOptions kotsadmtypes.KotsadmOptions) (*appsv1.Deployment, error) {
-	kotsadmTag := kotsadmversion.KotsadmTag(kotsadmtypes.KotsadmOptions{}) // default tag
-	image := fmt.Sprintf("kotsadm/minio:%s", kotsadmTag)
+	image := "minio/minio:RELEASE.2021-04-06T23-11-00Z"
```

Comment:
@salahalsaleh I think this is the equivalent of the original code, but how does this work with private registries and airgap?

Reply:
@divolgin https://github.com/replicatedhq/kots/pull/1802/files?w=1#diff-3ee38938c503a8ac3c5e47bf4596b2936ef10e22475a40a457123f20844985dcR249